source/octf/utils/table/internal/TableMap.h

```diff
@@ -17,51 +17,57 @@
 namespace octf {
 namespace table {
 
+/**
+ * @brief Cells' Map for the magic table (Table)
+ *
+ * This class is the map of cells keyed by addresses [row, column]. In addition
+ * it keeps association for rows and columns, returns Table iterators.
```

Comment:
To me "association for row and columns"  doesn't say much, I would say it is - an association of column/row names and their indexes.